controller/Review.php

```diff
@@ -109,7 +112,9 @@ public function index(): void
             'execution' => $execution->getIdentifier(),
             'delivery' => $delivery->getUri(),
             'show-score' => (int) $finder->getShowScoreOption($launchData),
-            'show-correct' => (int) $finder->getShowCorrectOption($launchData)
+            'show-correct' => (int) $finder->getShowCorrectOption($launchData),
+            'display-section-titles' => (int) $testRunnerPluginsConfig['review']['displaySectionTitles'] || true,
+            'review-layout' => $testRunnerPluginsConfig['review']['reviewLayout']
```

Comment:
I've switched to using `ltiTestReview` config (added a default config file and 2-way migration) instead of `taoQtiTest`.

I've also implemented the LTI params `review_layout` and `section_titles` (because why not) to override the config.